### DISCUSSION TITLE
Static sponsors with dynamic

### DISCUSSION THREAD

**Discussion by xunholy** [JamesIves/github-sponsors-readme-action#184] [Q&A] [answered]
Hi, I would like to have static with dynamic sponsors or have a way to retain sponsors in the list if they cancel their sponsorship.

My issue was when setting up the static sponsors having the marker breaks it onto a new line whereas I'd like it to appear as if it were all in the same line. Also AFAIK it clears everything between the markers - does this mean cancelled sponsors would no longer appear?

**Comment by JamesIves** [comment]
As of right now this action will remove sponsors who cancel their subscription as this pulls directly from the sponsorship API.

**Reply by Griefed** [reply]
Late to the party, but thanks for letting us know. Was hoping this was a feature. :D

I've been using your action for my README for quite a while, as it works perfectly well using [nektos/act](https://github.com/nektos/act). This way, I can update my README from my GitLab instance and mirror the repo to various destinations. Synced and updated READMEs everywhere!

**Reply by JamesIves** [reply, marked as answer]
Heads up, you can now set `active-only: false` to show _all_ sponsors regardless of their status. The `active-only` field will default to `true` which results in the current behavior!
